ide/che-core-ide-templates/src/main/resources/samples.json

```diff
@@ -989,6 +1044,69 @@
       "spring boot"
     ]
   },
+  {
+    "name": "spring-boot-gs-rest-service",
+    "displayName": "spring-boot-gs-rest-service",
+    "path": "/gs-rest-service",
+    "description": "Spring Boot Guide to build a RESTful web service ",
```

Comment:
Can you, please, remove space character in the end of ```spring-boot-gs-rest-service``` project description? Our selenium tests have some problem with it.